Formula/pspg.rb

```diff
@@ -0,0 +1,27 @@
+class Pspg < Formula
+  desc "Unix pager optimized for psql"
+  homepage "https://github.com/okbob/pspg"
+  url "https://github.com/okbob/pspg/archive/0.5.tar.gz"
+  sha256 "754d1e380d072517e9bc2c3c38785e2f19a9f927f061de9a646fd1094baa204e"
+  head "https://github.com/okbob/pspg.git"
+
+  depends_on "ncurses"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats; <<~EOS
+    Add the following line to your psql profile (e.g. ~/.psqlrc)
+         \\setenv PAGER pspg
+         \\pset border 2
+         \\pset linestyle unicode
+    EOS
+  end
+
+  test do
+    assert_match("pspg-#{version}", shell_output("#{bin}/pspg --version").chomp)
```

Comment:
You're gonna get asked for a functionality test beyond a simple version check by @ilovezfs, so, if you can, start thinking of one 😉.

Reply:
It's… a pager. Kind of hard to test since most of the interaction is e.g. via keyboard (arrow keys, `vim` keybindings, etc.)?

I looked at e.g. `nano`, which calls `--version` and doesn't even check that its output is sane…

🤷‍♂️

Reply:
Yeah, appreciate that. Just thought I'd throw it out there in case you had ideas. _Generally_ new formulae are expected to adhere to a higher standard than existing formulae.

Reply:
```
  test do
    ENV["TERM"] = "xterm"
    assert_match /\e\[\?1049.*\e>moo/, pipe_output("#{bin}/pspg -F", "moo", 0)
  end
```
maybe

Reply:
My shell-fu was too weak to get there (I was worried about repeatability). I can write something up like that (indeed I already had a mini-test that wrote out a sample header/row file and printed it using `pspg`.

Reply:
As long as the match is minimal it shouldn't need much tweaking over time.